index.html

```diff
@@ -214,6 +222,25 @@
                      function show_p() {
                          document.getElementById("pa").innerHTML = path;
                     }
+                       var clickmilestone_event=document.getElementById('Milestone');
+        clickmilestone_event.addEventListener('click',function(){
+          
+            const wrapper = document.createElement('div');
+wrapper.innerHTML = "<img src='https://previews.123rf.com/images/betka82/betka821710/betka82171000014/88096156-flame-icon-vector-fire-vector-fire-on-a-white-background.jpg' style='width:55px;style='float:left;display:flex;flex-direction:row;'><span style='color:black;font-size:30px;'><b style=''>1000</b></span><img src='https://previews.123rf.com/images/betka82/betka821710/betka82171000014/88096156-flame-icon-vector-fire-vector-fire-on-a-white-background.jpg' style='width:55px;'>"
```

Comment:
what if in future the path is changed its better to add them here in source code I think

Reply:
yes i think image must be saved to root directory in repo and then the local address

Reply:
check images folder add it there and update the PR

Reply:
Done !!! But My PR for image is seperate is it possible to merge New PR with old PR